.github/workflows/weekly.yml

```diff
@@ -20,13 +20,14 @@ name: Nightly
 on:
   schedule:
     - cron: "0 0 * * 0"
+  workflow_dispatch:
 jobs:
   compile_and_test:
     if: github.repository_owner == 'apache'
     strategy:
       matrix:
-        flink: [1.16-SNAPSHOT, 1.17-SNAPSHOT]
+        flink: [1.17-SNAPSHOT, 1.18-SNAPSHOT]
```

Comment:
Then we'll know for sure that the tests will be failing, since the Archunit upgrade in 1.17 is a breaking change compared to Flink 1.16. Then we need to consider also disabling the Archunit tests in the nightly builds (which might not be a bad idea)